Dockerfile

```
@@ -31,6 +31,8 @@ RUN apt-get update && apt-get install -y \
     build-essential libglib2.0-0 && \
     rm -rf /var/lib/apt/lists/*
 
+RUN apt-get update && apt-get install -y git
```

```suggestion
RUN apt-get install -y git
```

We don't need to update again since we call update in line 30. We could consolidate the git install with the others, but given that we will eventually get rid of the git dependency I'm ok with keeping it as a separate line.